sklearn/model_selection/tests/test_split.py

```diff
@@ -2054,3 +2054,19 @@ def test_no_group_splitters_warns_with_groups(cv):
 
     with pytest.warns(UserWarning, match=msg):
         cv.split(X, y, groups=groups)
+
+
+def test_Stratified_mandatory_y_argument():
+    x = ["a"] * 100
+    # y = [0] * 90 + [1] * 10
+    # Variable 'y' is required for Stratified classes split function.
+    # Testing the split without 'y', shoudl raise TypeError.
```

Comment:
I think you can remove the comment, the line about `y` is not useful and I feel the test is simple enough that it does not need any comment to explain it
```suggestion
```